src/stylesheets/components/_footer.scss

```diff
@@ -17,4 +17,10 @@
     margin-bottom: 0;
     border-bottom: 0;
   }
+
+  // Fix the visually hidden 'Support links' heading causing Safari to pad out
+  // the bottom of the page or... something.
```

Comment:
Should this link to the issue with the screenshot?

Reply:
The commit message references it which I think is _probably_ enough?